docs/installation/octopus-in-container/octopus-server-container-linux.md

```diff
@@ -53,10 +53,9 @@ Read Docker [docs](https://docs.docker.com/engine/reference/commandline/run/#pub
 |  Port       |    |
 | ------------- | ------- |
 |**8080**| Port for API and HTTP portal |
+|**443**| SSL Port for API and HTTP portal |
 |**10943**|Port for Polling Tentacles to contact the server|
 
-_The Octopus Server container does not currently support HTTPS, however, this should be available sometime in the future_
```

Comment:
I'm not sure we've actually made this work? Looking at the [dockerfile](https://github.com/OctopusDeploy/OctopusDeploy/blob/master/docker/linux/Dockerfile) it doesn't seem to allow it to be configured?
Users can build their own container on top of ours, but not sure we support it out of the box?

Reply:
From @cvandal on Slack 

> we don’t use this any more since we moved from dedicated static ip addresses to a pool of static ip addresses, but here’s the code i wrote to support port 443, force ssl, and import the certificate: https://github.com/OctopusDeploy/OctopusDeploy/pull/4734/files